app/assets/javascripts/discourse/components/d-editor.js.es6

```diff
@@ -616,6 +616,16 @@ export default Ember.Component.extend({
     Ember.run.scheduleOnce("afterRender", () => $textarea.focus());
   },
 
+  paste: function(e) {
+    var items = (e.clipboardData || e.originalEvent.clipboardData).items;
+
+    $.each(items, function(i, d) {
```

Comment:
In ES6, you can use [forEach](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) instead of `$.each`

Reply:
O you should probably have a look at [arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) as well.